set-client-version.py

```diff
@@ -102,6 +102,24 @@ def set_installshield(version):
                     line = f'		<row><td>ProductVersion</td><td>{version}</td><td/></row>\n'
                 f.write(line)
 
+def set_snapcraft(version):
+    with open('snap/snapcraft.yaml','r') as f:
+        lines = f.readlines()
+    with open('snap/snapcraft.yaml','w') as f:
+        for line in lines:
+            if line.startswith('version:'):
+                line = f'version: "{version}"\n'
+            f.write(line)
+
+def set_boinc_desktop(version):
```

Comment:
Since this is hardcoded to change snap desktop file, please rename it to `set_snap_boinc_desktop`

Reply:
done.